docs: add preact devtools option (#8164)
withastro/docs: src/content/docs/en/guides/integrations-guide/preact.mdx

Currently, the `compat` option only works for React libraries that export code as ESM. If an error happens during build-time, try adding the library to `vite.ssr.noExternal: ['the-react-library']` in your `astro.config.mjs` file.
:::

### devtools

> You can enable [Preact devtools](https://preactjs.github.io/preact-devtools/) in development by setting `devtools: true`.

Reviewer: ```suggestion
You can enable [Preact devtools](https://preactjs.github.io/preact-devtools/) in development by setting `devtools: true` in your `preact()` integration config:
```

Currently, the `compat` option only works for React libraries that export code as ESM. If an error happens during build-time, try adding the library to `vite.ssr.noExternal: ['the-react-library']` in your `astro.config.mjs` file.
:::

### devtools

<p><Since v="3.3.0" /></p>

You can enable [Preact devtools](https://preactjs.github.io/preact-devtools/) in development by setting `devtools: true` in your `preact()` integration config:

```js title="astro.config.mjs"
import { defineConfig } from 'astro/config';
import preact from '@astrojs/preact';

export default defineConfig({
  // ...
  integrations: [preact({ devtools: true })],
});

> 

Reviewer: ``````suggestion
```

``````

Currently, the `compat` option only works for React libraries that export code as ESM. If an error happens during build-time, try adding the library to `vite.ssr.noExternal: ['the-react-library']` in your `astro.config.mjs` file.
:::

### devtools

> <p><Since v="3.3.0" /></p>

Reviewer: ```suggestion
<p><Since pkg="preact" v="3.3.0" /></p>
```
Let's see if this works!

> <p><Since pkg="preact" v="3.3.0" /></p>

Reviewer: ```suggestion
<p><Since pkg="@astrojs/preact" v="3.3.0" /></p>
```

> You can enable [Preact devtools](https://preactjs.github.io/preact-devtools/) in development by setting `devtools: true` in your `preact()` integration config:

Reviewer: ```suggestion
You can enable [Preact devtools](https://preactjs.github.io/preact-devtools/) in development by passing an object with `devtools: true` to your `preact()` integration config:
```
Just looked at the section above, and this is closer to the wording used there, so I'll change that here and in the other PR!
